packages/react/src/components/Card/Card.stories.tsx

```diff
@@ -44,167 +42,351 @@ export const Preview: Story = (args) => (
 );
 
 Preview.args = {
-  shadow: 'medium',
-  variant: 'neutral',
+  color: 'neutral',
 };
 
+export const LinkCard: Story = (args) => (
+  <div
+    style={{
+      display: 'grid',
+      gap: 'var(--fds-spacing-4)',
+      gridTemplateColumns: 'repeat(2, 400px)',
+    }}
```

Comment:
Can we make this `flex`? 😄 
<img width="911" alt="image" src="https://github.com/digdir/designsystem/assets/14057205/494286cb-8ae7-4868-8e0a-57d379e7cf8a">

Reply:
What wrong with `grid`? Don't we want to limit it to two columns and cards to be 400px?

Reply:
```css
display: flex;
gap: var(--fds-spacing-4);
width: calc(800px + var(--fds-spacing-4));
flex: 1 1 0px;
```

Sure, I'm just irritated by the vertical gap 😄

Reply:
Lets just keep the styling simple as it just serves as an example 😅